benchmark/fixtures/require-cachable.js

```diff
@@ -3,7 +3,7 @@
 const list = require('internal/bootstrap/cache');
 const {
   isMainThread
-} = require('worker_threads');
+} = require('internal/worker');
```

Comment:
Don’t we want to use public API whenever possible?

Reply:
This should be no longer necessary after the `--experimental-worker` flag is removed.